### ISSUE TITLE
规划

### ISSUE THREAD

**Issue by kayang-potato** [kayang-potato/SuperKiller#1]
# 数据处理
  -  接口输出文档化，如数据处理中的数据观察、画图等，都可以直接文档化，方便易用；
# 机器学习
  - 回归算法
    -  Ridge、 Lasso、ElasticNet、LinearRegression、SVR、DecisionTreeRegressor、BayesianRidge 、RandomForestRegressor、XGBRegressor、GradientBoostingRegressor、Adaboost Regressor、LGBMRegressor;
    - 不常用：Huber回归、KernelRidge、CatBoostRegressor、ExtraTreesRegressor、StackingRegressor、VotingRegressor、StackingCVRegressor；
  - 分类算法
    - LR、SVM、RandomForestClassifier、LightGBM、Xgboost、GaussianNB、Perceptron、DT、KNN、GradientBoostingClassifier、SGDClassifier、Naive Bayes、Stochastic Gradient Descent、RidgeClassifier、ExtraTreesClassifier、AdaBoostClassifier、LinearSVC、ExtraTreesClassifier；
  - 无监督算法
    - kmeans等；
  - 时序算法
    - HMM等；
  - python框架
    - 数据分析（代表性数据、散点图、曲线图、密度图、热力图等）
    - 数据处理（数据清洗、归一化、扩充等）
    - 特征工程（筛选显著特征，相关性分析等）
    - 模型参数（参数初始化，网格搜索参数设置等）
    - 模型训练
    - 模型预测
    - 模型评估
    - Tip
      - 模型训练和预测需要大量的头文件，可以将这些整合至一个文件中（包括数据分析处理以及模型评估等都可以在不同函数里进行引入）；
